Sources/BraintreePayPal/BTPayPalVaultEditRequest.swift

```diff
@@ -1,20 +1,40 @@
 import Foundation
 
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
 /// Options for the PayPal edit funding instrument flow
 /// - Warning: This feature is currently in beta and may change or be removed in future releases.
-public struct BTPayPalVaultEditRequest {
+public class BTPayPalVaultEditRequest {
 
+    var riskCorrelationID: String?
+    let hermesPath: String = "v1/paypal_hermes/generate_edit_fi_url"
     private let editPayPalVaultID: String
-    private let merchantAccountID: String?
+
+    // MARK: - Static properties
+
+    static let callbackURLHostAndPath: String = "onetouch/v1"
 
     /// Initializes a PayPal Edit Request for the edit funding instrument flow
     /// - Parameters:
     ///   - editPayPalVaultID: Required: The `edit_paypal_vault_id` returned from the server side requests,
     ///   `gateway.payment_method.find("payment_method_token")` or `gateway.customer.find("customer_id")`
     ///   - merchantAccountID: Optional: ID of the merchant account; if one is not provided the default will be used
     /// - Warning: This feature is currently in beta and may change or be removed in future releases.
-    public init(editPayPalVaultID: String, merchantAccountID: String? = nil) {
+    public init(editPayPalVaultID: String) {
         self.editPayPalVaultID = editPayPalVaultID
-        self.merchantAccountID = merchantAccountID
+    }
+
+    public func parameters() -> [String: Any] {
+        var parameters: [String: Any] = [:]
+
+        parameters["edit_paypal_vault_id"] = editPayPalVaultID
+
+        parameters["return_url"] = BTCoreConstants.callbackURLScheme + "://\(BTPayPalRequest.callbackURLHostAndPath)success"
+        parameters["cancel_url"] = BTCoreConstants.callbackURLScheme + "://\(BTPayPalRequest.callbackURLHostAndPath)cancel"
+
+        parameters["risk_correlation_id"] = riskCorrelationID
+        return parameters
```

Comment:
Since we already know the values we can construct the dictionary like so:
```suggestion
        [
            "edit_paypal_vault_id": editPayPalVaultID,
            "return_url": BTCoreConstants.callbackURLScheme + "://\(BTPayPalRequest.callbackURLHostAndPath)success",
            "cancel_url": BTCoreConstants.callbackURLScheme + "://\(BTPayPalRequest.callbackURLHostAndPath)cancel",
            "risk_correlation_id": riskCorrelationID
        ]
```

Reply:
It looks like we are defining `callbackURLHostAndPath` but just pulling it here from `BTPayPalRequest`. Which do we want to use?

Reply:
oh yeah, what I did doesn't make much sense. should use the local property.